Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -49,6 +29,14 @@ public static void HookEvents(IVBE vbe)
                 }
                 _threadId = User32.GetWindowThreadProcessId(mainWindowHwnd, IntPtr.Zero);
                 _eventHandle = User32.SetWinEventHook((uint)WinEvent.Min, (uint)WinEvent.Max, IntPtr.Zero, _eventProc, 0, _threadId, WinEventFlags.OutOfContext);
+
+                var subclasses = mainWindowHwnd.ChildWindows()
+                    .Where(hwnd => SubclassManager.IsSubclassable(hwnd.ToWindowType()));
```

Comment:
Thinking that should be made a method on the `SubclassManager` to encapsulate the filters, so the calling code becomes simply `SubclassManager.AttachBulk(mainWindowHwnd.ChildWindows())`

Reply:
Nice. Consider it done.